src/providers/postgres/qgspostgresproviderconnection.h

```diff
@@ -66,6 +67,8 @@ class QgsPostgresProviderConnection : public QgsAbstractDatabaseProviderConnecti
     void dropTablePrivate( const QString &schema, const QString &name ) const;
     void renameTablePrivate( const QString &schema, const QString &name, const QString &newName ) const;
 
+    class QgsPostgresConn *pgconn;
```

Comment:
```suggestion
    class QgsPostgresConn *mPgConn;
```

Reply:
Thanks, will fix. Currently checking the QGIS Postgres provider tests to see if they (still) run.

@elpaso If these run OK I intend to drop commits number 3 and 4 of this PR and just leave commits 1, 2 and 5.
Is that OK with you?